compiler/src/iree/compiler/Codegen/Common/CPU/CPUMaterializeEncodingPass.cpp

```diff
@@ -304,11 +320,20 @@ std::unique_ptr<OperationPass<func::FuncOp>>
 createCPUMaterializeEncodingPass() {
   return std::make_unique<CPUMaterializeEncodingPass>();
 }
+std::unique_ptr<OperationPass<func::FuncOp>>
+createCPUMaterializeEncodingPass(IREE::HAL::ExecutableTargetAttr targetAttr) {
```

Comment:
Maybe just use a default value for the attr to be null (and the `ArrayRef<..>` to be empty below) to avoid duplicate entry points.